docs/cmake-settings.md

```diff
@@ -0,0 +1,66 @@
+# Configure CMake tools
+
+CMake Tools supports a variety of settings that can be set at the user, or workspace, level via VSCode's `settings.json` file. This topic  covers the available options and how they are used.
+
+Options that support substitution, in the table below, allow variable references to appear in their strings. See [variable substitution](#variable-substitution), below, for more information about variable expansion.
+
+## CMake settings
+
+|Setting  |Description | Default value | Supports substitution |
```

Comment:
There are two commands we've added since taking over the extension that aren't captured here:
cmake.autoSelectActiveFolder | If 'false', your active folder will only change if you manually run the CMake: Select Active Folder command | 'true' | no
cmake.cmakeCommunicationMode | Specifies the protocol used to communicate between the extension and CMake | 'automatic' | no

Reply:
Added